tasks/main.yml

```diff
@@ -71,7 +71,7 @@
       # We need to use shell instead of command because of the above --login flag, since
       # nix-env isn't in the default path.
       # noqa command-instead-of-shell
-      ansible.builtin.shell: "nix-env -iA nixpkgs.nixFlakes"
+      ansible.builtin.shell: "nix-env -iA nixpkgs.nixVersions.stable"
```

Comment:
The task still mentions "Nix Flakes package"

Reply:
It's an alias, so isn't this technically still correct?

Reply:
Quoting https://github.com/NixOS/nixpkgs/pull/123898

> There no longer is a separate `nixFlakes` package, flake support has been in master for a while.

What we are installing is the Nix stable package.

Reply:
Fair enough! Fixed in 5d52dc6.